reference/joint.md

```diff
@@ -2,8 +2,8 @@
 
 ```
 Joint {
-  SFNode jointParameters NULL # a joint parameters node
-  SFNode endPoint        NULL # Solid or SolidReference
+  SFNode jointParameters NULL   # {JointParameters, PROTO}
```

Comment:
To be consistent JointParameters (or derived) -> depending on the type of joint JointParameters or HingeJointParameters is allowed